Dockerfile.linux_amd64

```diff
@@ -0,0 +1,13 @@
+FROM amd64/ubuntu
+RUN apt-get update && \
+    apt-get install -y protobuf-compiler git cmake gcc g++
+WORKDIR /workdir
+COPY . /workdir
+RUN git submodule init
+RUN git submodule update
```

Comment:
This step should happen in the local folder instead of in the docker (less during the build stage of the image).